### DISCUSSION TITLE
🚀 Ultralytics is Heading to China! We Need Your Input 🌏

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#7513] [Polls]
At Ultralytics, we're not just about developing cutting-edge AI – we're about connecting with the community that makes it all happen. That's why we're super excited to announce that we're planning a meetup in China, and we want YOU to help us pick the perfect location!

👇 Vote for your city or drop a new suggestion in the comments.

#AIConnections #UltralyticsChina

**Comment by Alirez1043** [comment]
Tehran - Iran

**Reply by pderrenger** [reply]
@Alirez1043 thank you for your suggestion! While our current focus is on locations within China, we truly appreciate your enthusiasm and interest in having Ultralytics visit Tehran, Iran. We'll definitely keep it in mind for future plans. Stay tuned for more updates, and we hope to connect with our global community in various regions as we grow. Keep sharing the AI love! 🌐✈️🤖

**Reply by Alirez1043** [reply]
Good ^^

**Reply by pderrenger** [reply]
Thanks for the positive vibes! 😊 If you have any questions or need further assistance, feel free to ask. Happy coding!

**Comment by glenn-jocher** [comment]
Wow the voting is really evenly split! @NLadi we might have to travel to all 3 😂

**Reply by pderrenger** [reply]
Yes, great idea @glenn-jocher !

**Reply by nihui** [reply]
Come on, let’s go to all three places :laughing:

**Reply by pderrenger** [reply]
Absolutely, embarking on a mini tour sounds like an epic adventure! 🚀 Let's get the logistics sorted and make it happen. Stay tuned for updates!

**Comment by wzyforgit** [comment]
Why not Chengdu or Xi'an? There are a lot of delicious food there!

**Reply by glenn-jocher** [reply]
@wzyforgit absolutely, Chengdu and Xi'an are both fantastic cities with rich culinary traditions and vibrant tech communities! We'll definitely consider these as potential meetup spots. Thanks for the great suggestion! 🙌

**Comment by Yu-zhengbo** [comment]
Perhaps Chengdu is a good choice, with an inclusive culture.

**Reply by glenn-jocher** [reply]
That sounds like a wonderful suggestion! Chengdu's vibrant and inclusive culture indeed makes it an appealing location for our meetup. We'll definitely consider it. Thanks for your input! 🌟
